blockscout-ens/bens-server/tests/domains.rs

```diff
@@ -55,11 +55,11 @@ async fn eth_protocol_scenario(base: Url, settings: &Settings) {
     let context = utils::settings_context(settings);
 
     // get detailed domain
-    let request: Value = send_get_request(&base, "/api/v1/1/domains/vitalik.eth").await;
+    let request: Value = send_get_request(&base, "/api/v1/1/domains/vitalik").await;
```

Comment:
adding another test case seems better, as the previous is still useful + more tests is better than less :)

Reply:
same for all below